src/index.html

```diff
@@ -1,13 +1,282 @@
+                alt="phone"
+                class="categories__image categories__image--img-before-tablet"
+              >
+            </a>
+            <a href="#" class="categories__products-6">
+              <img
+                src="./images/categories/charger.svg"
+                alt="charger"
+                class="categories__image"
+              >
+            </a>
+          </div>
+          <h3 class="categories__description">Accessories</h3>
+        </article>
+        <a href="#" class="categories__view-all">View all</a>
+      </section>
+
+      <section class="about-us" id="about-us">
+        <img
+          src="./images/about-us/bg-before-about-us.svg"
+          alt="background"
+          class="about-us__bg"
+        >
+        <div class="about-us__container">
+          <h2 class="section-title about-us__title">About us</h2>
+          <p class="about-us__description">We’re a London-based tech company on a mission to remove barriers between people and technology.
+            <br>
+            <br>
+            To do this, we’re crafting intuitive, flawlessly connected products that improve our lives without getting in the way.
+            <br>
+            No confusing tech-speak. No silly product names. Just artistry, passion and trust. And products we’re proud to share with our friends and family. Simple.
+            <br>
+            <br>
+            Because beautiful tech shouldn’t be complicated.
+          </p>
+
+        </div>
+      </section>
+
+      <section class="contact" id="contact">
+        <div class="contact-container">
+          <div class="contact-container__form">
+            <h2 class="section-title">Contact us</h2>
+            <form
+              action="http://localhost:1234/api"
+              method="post"
+              class="contact__form"
+            >
+              <div class="contact__form-field">
+                <input
+                  type="text"
+                  name="name"
+                  placeholder="Name"
+                  class="contact__form-field__element contact__form-field__input"
+                >
+              </div>
+              <div class="contact__form-field">
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="Email"
+                  class="contact__form-field__element contact__form-field__input"
+                >
+              </div>
+              <div class="contact__form-field">
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="contact__form-field__element contact__form-field__textarea"
+                  ></textarea>
+              </div>
+              <a href="#" class="contact__button">Send</a>
+            </form>
+
+          </div>
+          <div class="contact-container__company-datas">
+            <div class="contact__company-data">
```

Comment:
The text color of phone, email and address should change on hover [Checklist #4](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Change%20text%20color%20on%20hover%20for%20phone%2C%20email%2C%20and%20address)